installers/pgo-installer/bin/pgo-install.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+ansible-playbook -i /ansible-inventory/inventory --tags=$1 /ansible/main.yml
```

Comment:
Do we want to assume that `ansible-playbook` is on the `$PATH`?

Reply:
Possibly not, but we are creating the image so I would imagine we know where it is.

I'm honestly not sold on keeping this script anyway. @abrightwell what are your thoughts on that? I could pass the whole ansible line into the job instead of using this.

Reply:
I mean this is encapsulated in the container?

It would not hurt to have a `/usr/bin/env ansible-playbook`